### DISCUSSION TITLE
2FA is draconian control on freedom.

### DISCUSSION THREAD

**Discussion by PalpatineSWU** [community/community#113402] [Other Feature Feedback, Questions, & Ideas]
I simply don't care what your implying this change is for, it's clearly just to impose more control on free citizens.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @PalpatineSWU! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by o-b-o** [reply]
Y'all just go with the flow eh?

**Comment by Qix-** [comment]
This is a new take on 2fa, care to explain how this asserts control...?

**Reply by o-b-o** [reply]
Implied consent. Take away choice from the user because devs know best. It's not like you get any say, they will go ahead. But ok, github, it's been real, but you suck now. '
Restrict account actions'? What does that mean? 
Can I still delete my code? Close my account if I don't agree to your changes?

**Reply by Qix-** [reply]
Wat. This is the most needlessly staunch take. 2FA is required a lot of places. It is generally considered defacto better security.

Can you name any _real_ downsides? Calling it draconian is so melodramatic I don't even know where to begin.

**Comment by Andre601** [comment]
I'm unsure what exactly happened to you that you consider 2FA bad here...

Are you required to enable it for certain actions now? It would be great if you could provide some additional information and context here, so that everyone understands the issue.

I for my part appreciate the 2FA of GitHub, because unlike other platforms and services does GitHub regularely ask you about the up-to-date state of your 2FA settings and also use it on various actions one may consider dangerous, such as deleting an entire repository...

**Reply by o-b-o** [reply]
it's not asking you 
it's telling you

**Reply by Andre601** [reply]
This doesn't answer my question. Especially not by the OP which I've been asking here. So unless you are the OP - in which case you would violate GitHub's ToS as only one account per user is allowed - would I kindly ask you to not reply to questions not aimed at you.

**Reply by o-b-o** [reply]
Oh you're a kind one. Thanks for being so kind, friend.
